src/services/scheduler.ts

```diff
@@ -194,7 +201,7 @@ export async function start(dbId: string, dbUser: string) {
           try {
             const recordCount = await iasql.getDbRecCount(conn);
             const rpcCount = await iasql.getRpcCount(conn);
-            await MetadataRepo.updateDbCounts(dbId, recordCount, rpcCount);
+            await MetadataRepo.updateDbCounts(dbId, recordCount, undefined, rpcCount);
```

Comment:
TODO to remove this as well

Reply:
I did not add it exactly here but in the function that needs to be updated.